app/models/user.rb

```diff
@@ -20,6 +20,11 @@ class User < ActiveRecord::Base
   has_many :outgoing_delegations, class_name: 'ApprovalDelegate', foreign_key: 'assigner_id'
   has_many :outgoing_delegates, through: :outgoing_delegations, source: :assignee
 
+  has_many :incoming_delegations, class_name: 'ApprovalDelegate', foreign_key: 'assignee_id'
+  has_many :incoming_delegates, through: :incoming_delegations, source: :assigner
```

Comment:
I used them in order to diagnose the original story description. Since they seemed to be missing as symmetrical associations for `outgoing_delegations` I left them there as useful later on (as when exposed via /admin e.g.).